pkg/client/grafanacloud/client.go

```diff
@@ -0,0 +1,80 @@
+// Package grafanacloud provides an interface to the Grafana Cloud API.
+package grafanacloud
```

Comment:
This is out of scope but we really should start consolidating our GCom clients :slightly_smiling_face: